include/gmsh.tcc

```diff
@@ -1,6 +1,6 @@
 //! Read vertex id and coordinates in GMSH
+//! \tparam Tdim Dimension, Tvert Number of vertices in element
```

Comment:
`Tvert` should be in the next line. Use `doxygen` to check if the comments are generated properly.